README.md

```diff
@@ -1 +1,99 @@
-Coming soon…
+# UI Component Best Practices <!-- omit in toc -->
+
+A bunch of best practices to help achieve User Interface (UI) components, or
+presentational components, that are:
+
+- accessible
+- consumable
+- portable
+- reusable
+- and robust.
+
+## Table of Contents <!-- omit in toc -->
+
+- [Caveats](#caveats)
+- [The tl;dr Version](#the-tldr-version)
+- [Components Cannot Apply External Spacing](#components-cannot-apply-external-spacing)
+  - [The Rule](#the-rule)
+  - [Why?](#why)
+    - [Why `margin` Specifically?](#why-margin-specifically)
+  - [The Exception(s)](#the-exceptions)
+  - [More Reading](#more-reading)
+
+## Caveats
+
+1. These best practices target components that are part of an organisation's component library or broader design system where the design is known.
+2. All code snippets are from the [React library](https://reactjs.org/).
```

Comment:
Do you mean you are using code snippets from their site, or that they are using JSX?

Reply:
I updated it to make it clearer.